### PR TITLE
Allow `tags` on `span.contexts`

### PR DESCRIPTION
**Pull request by simitt**
implements elastic/apm-server#1152 

cc @felixbarny

### CODE REVIEW

**Comment by simitt** [comment]
@watson the indentation in the generated files has never been accurate.

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -14,6 +14,7 @@ https://github.com/elastic/apm-server/compare/6.3\...master[View commits]
 
 - Change `frontend` to `rum` in config file, but still support `frontend` for backwards compatibility {pull}1155[1155].
 - Support `rum` and `client-side` endpoint for RUM for backwards compatibility {pull}1155[1155].
+- Allow sending `tags` for `spans`, that get indexed in ES {pull}1156[1156].
```
Comment (roncohen) [inline CHANGELOG.asciidoc]: accidental double entry

**Comment by simitt** [comment]
Closing this according to https://github.com/elastic/apm-server/issues/1152#issuecomment-406608916

**Comment by simitt** [comment]
I reopened this PR, as according to offline discussions we will allow `span.tags` to use the current ES schema for `transaction.tags`. 
@roncohen or @jalvz could you please re-review.

**Comment by jalvz** [comment]
lgtm. just move the changelog line from 6.4 section to head

File: docs/common-problems.asciidoc [hunk 2]
```diff
@@ -177,3 +178,37 @@ This is not a SSL problem. Make sure that Logstash is running and that there is
...
 destination host.
+
+[[field-limit-exceeded]]
+[float]
+=== Field limit exceeded
+
+When adding too many distinct tag keys on a transaction or span, 
+you risk creating a link:{ref}/mapping.html#mapping-limit-settings[mapping explosion].
+
+For example,
```
Comment (jalvz) [inline docs/common-problems.asciidoc]: i know is not introduced here, but i'd appreciate if we stop doing line breaks systematically after punctuation marks. it makes docs harder to read for no good reason.

Comment (simitt) [reply docs/common-problems.asciidoc]: I'd like to defer this discussion to general docs discussions and not just break with our internal guidelines here.

**Pull request merged**